scalardb-server-sample/README.md

```diff
@@ -94,39 +137,39 @@ You can create a transaction-enabled schema by setting `transaction` to true. (T
 }
 ```
 
-You can apply the schema in the same way as above. Note that `--coordinator` is specified to create the coordinator table needed for transactions.
+You then apply the schema with the following command.
+```shell
+$ java -jar scalardb-schema-loader-<version>.jar --config scalardb-client.properties --schema-file emoney-storage.json
+```
 
+### Run the sample with Storage API
+- Charge `1000` to `user1`:
+```shell
+$ ./gradlew -Pstorage run --args="-action charge -amount 1000 -to user1"
 ```
-java -jar scalardb-schema-loader-<version>.jar --config scalardb-client.properties --schema-file src/main/resources/emoney-transaction-schema.json --coordinator
+
+- Charge `0` to `merchant1` (Just create an account for `merchant1`):
+```shell
+$ ./gradlew -Pstorage run --args="-action charge -amount 0 -to merchant1"
 ```
 
-## Run the sample
-Scenario:
-  - Charge the user `user1` with an amount of 10.
-  - Charge the user `user2` with an amount of 20.
-  - `user1` pays an amount of 5 to `user2`.
-
-Run
-  - Storage mode
-   ```
-   # charge user1 with an amount of 10
-   ./gradlew run --args="cmd storage charge -a 10 -u user1"
-
-   # charge user2 with an amount of 20
-   ./gradlew run --args="cmd storage charge -a 20 -u user2"
-
-   # pay 5 from user1 to user2
-   ./gradlew run --args="cmd storage pay -a 5 -f user1 -t user2"
-   ```
-
-  - Transaction mode
-   ```
-   # charge user1 with an amount of 10
-   ./gradlew run --args="cmd transaction charge -a 10 -u user1"
-
-   # charge user2 with an amount of 20
-   ./gradlew run --args="cmd transaction charge -a 20 -u user2"
-
-   # pay 5 from user1 to user2
-   ./gradlew run --args="cmd transaction pay -a 5 -f user1 -t user2"
-   ```
+- Pay `100` from `user1` to `merchant1`:
+```shell
+$ ./gradlew -Pstorage run --args="-action pay -amount 100 -from user1 -to merchant1"
+```
+
+- Get the balance of `user1`:
+```shell
+$ ./gradlew -Pstorage run --args="-action getBalance -id user1"
+```
+
+- Get the balance of `merchant1`:
+```shell
+$ ./gradlew -Pstorage run --args="-action getBalance -id merchant1"
+```
+
+## Clean up
+To stop Cassandra and Scalar DB Server, run the following command:
+```shell
+$ docker-compose -f docker-compose-cassandra.yml down
```

Comment:
```suggestion
$ docker-compose -f docker-compose-cassandra.yml down --volume
```
If we don't add `--volume` flag in the `docker-compose down` command, the `docker volume` that includes Cassandra data is not removed as follows.

```console
$ docker-compose -f docker-compose-cassandra.yml down
Stopping scalardb-server-1 ... done
Stopping cassandra-1       ... done
Removing scalardb-server-1 ... done
Removing cassandra-1       ... done
Removing network scalar-network
```
```console
$ docker volume ls | grep scalardb-server-sample_cassandra-data
local     scalardb-server-sample_cassandra-data
```

I think we don't need to remain this `docker volume` since this is a just sample.
So, I think it is better to remove this `docker volume` in this section by adding the `--volume` flag.

```console
$ docker-compose -f docker-compose-cassandra.yml down --volume
Removing network scalar-network
WARNING: Network scalar-network not found.
Removing volume scalardb-server-sample_cassandra-data
```
```console
$ docker volume ls | grep scalardb-server-sample_cassandra-data
(No output)
```

Reply:
In the another samples, it seems that there is no definition of `docker volume` of each DB in the `docker-compose.yaml` (`docker volume` is not created).

https://github.com/scalar-labs/scalardb-samples/blob/main/scalardb-sample/docker-compose.yml
https://github.com/scalar-labs/scalardb-samples/blob/main/multi-storage-transaction-sample/docker-compose.yml

So, I think we can remove `docker volume` definition from `docker-compose.yaml` since we don't need to make data persistent, instead of adding `--volume` flag to `docker-compose down` command.

I think this way (don't create `docker volume`) is better since it is simple.

Reply:
Thank you for the suggestion! I just removed the unnecessary volume. Please take a look again when you have time!